RemotingSample/Assets/Prefabs/HandAxes.prefab

```diff
@@ -0,0 +1,413 @@
+%YAML 1.1
```

Comment:
Since we're using MRTK hands, we don't use this prefab, right?

Reply:
removed